src/NUnitFramework/testdata/SetUpFixtureData.cs

```diff
@@ -107,10 +106,10 @@ public static void RegisterEvent(string evnt)
         /// <param name="expectedEvents">The expected events.</param>
         public static void Verify(params string[] expectedEvents)
         {
+            int item = 0;
```

Comment:
Nice find

Reply:
Yes, it would only claim to fail on parameter 1, regardless what parameter failed.
Introduced 9 years ago to add a message in case of failure. I guess it never failed.